composer.json

```diff
@@ -4,14 +4,19 @@
     "description" : "Provide APCu support",
     "require": {
         "php"                            : ">=7.4",
-        "ext-apcu"                       : ">=4.0.7, <5.3"
+        "ext-apcu"                       : ">=4.0.7"
```

Comment:
:information_source: I don't see a [version greater than 5.1.*](https://pecl.php.net/package/APCu) and think this limit is unnecessary.